libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -573,6 +573,15 @@ class SRV_Channels {
         return channel_function(channel) == SRV_Channel::k_alarm_inverted;
     }
 
+    // return true if 32 channels are enabled
+    static bool have_32_channels() {
+#if NUM_SERVO_CHANNELS >= 17
+        return _singleton->enable_32_channels.get() > 0;
```

Comment:
can you go direct?
```suggestion
        return enable_32_channels.get() > 0;
```
Were a bit inconsistent in checking the singleton....

Reply:
No - it's a static function so can't access members directly.